scripts/drone/steps/lib.star

```diff
@@ -614,7 +614,7 @@ def test_a11y_frontend_step(ver_mode, port = 3001):
       Drone step.
     """
     commands = [
-        "yarn wait-on http://$HOST:$PORT",
```

Comment:
- we use node 14 in the docker-puppeteer docker image used to run pa11y
 - yarn 4 is not compatible with node 14
 - pa11y is not compatible with node 18 or 20
 - therefore, we use npx to run wait-on, avoiding yarn

Another solution to this would be to pre-install wait on (through npm) into the docker-puppeteer image, but this was easier.